### PR TITLE
Adding method to extract all "marker-file" properties

### PR DESCRIPTION
**Pull request by fdansv**
Ref #7, comes from cartodb/torque#80

cc @javisantana

### CODE REVIEW

File: lib/carto/renderer_js.js [hunk 1]
```diff
@@ -198,6 +202,12 @@ CartoCSS.prototype = {
         var layer = layers[key] = (layers[key] || {
           symbolizers: []
         });
+        for(var u = 0; u<def.rules.length; u++){
+            if(def.rules[u].name == "marker-file" || def.rules[u].name == "point-file"){
```
Comment (javisantana) [inline lib/carto/renderer_js.js]: use === whenever is possible

**Comment by javisantana** [comment]
:es:

**Comment by javisantana** [comment]
try to add a test in here: https://github.com/CartoDB/carto/blob/master/test/rendering_js.test.js

**Comment by fdansv** [comment]
:shipit: ?

**Comment by javisantana** [comment]
:es: :es:

**Pull request merged**